.github/workflows/e2e.yaml

```diff
@@ -64,12 +64,18 @@ jobs:
         export SHELL=/bin/bash
         make build image
     - name: Start KinD
-      uses: engineerd/setup-kind@v0.5.0
+      uses: helm/kind-action@v1.7.0
```

Comment:
Any special reason to change the action?

Reply:
Well, just two comments above would explain it 🤦 

note to self, read the PR conversation before reading the changes :P